### PR TITLE
Added an implementation of the change exercise

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: exercises/change/src/example/kotlin/Change.kt [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        issueRec(target, maxIndex)
+    }
+
+    private fun List<Int>.isNoCoinsResult() = (this == NO_COINS_INDICATOR)
+
+    private fun List<Int>.safeResult() = if(this.isNoCoinsResult()) listOf() else this
+
+}
+
+private object Memoizer {
```
Comment (sdavids13) [inline exercises/change/src/example/kotlin/Change.kt]: This doesn't seem to do anything.

**Comment by sdavids13** [comment]
@kytrinyx Can you add @geoand as a Kotlin track committer? He has been very helpful with providing additional exercises and I don't want to get in the way of that ;).

File: exercises/change/src/example/kotlin/Change.kt (hunk 1 above)
Comment (geoand) [inline exercises/change/src/example/kotlin/Change.kt]: You're right!

It forgot to remove it :(.

Will fix ASAP

**Comment by geoand** [comment]
@sdavids13 Thanks for the vote of confidence :)!

**Comment by kytrinyx** [comment]
Done! In the future you can do this from your account page on the exercism site (this is insufficiently documented).

**Comment by geoand** [comment]
Awesome!
Thanks a lot!

**Comment by kytrinyx** [comment]
Also: I got mixed up—you can add track mentors on the account page, not committers. Fixed.

**Pull request merged**